apps/views.py

```diff
@@ -424,15 +425,8 @@ def get(self, request, user, project, app_slug, data=[], wait=False, call=False)
             return HttpResponseForbidden()
 
         if app.slug == "minio":
-
-            def generate_password(len: int) -> str:
-                return "".join(
-                    secrets.choice(string.octdigits + string.ascii_uppercase + string.ascii_lowercase + string.digits)
-                    for i in range(len)
-                )
-
-            MINIO_USERNAME = generate_password(8)
-            MINIO_PASSWORD = generate_password(8)
+            MINIO_USERNAME = get_random_string(8)
+            MINIO_PASSWORD = get_random_string(8)
```

Comment:
I forgot to mention, that this is just styling issue. This type of casing should be used just for module level constants, so we should avoid doing that here

```suggestion
            minio_username = get_random_string(8)
            minio_password = get_random_string(8)
```

Reply:
Good point! I'll fix this. It needs to be changed in templates aswell

Reply:
Fixed!